models/bridges/bridges_bridge_transfers.sql

```diff
@@ -0,0 +1,128 @@
+{{ config(
```

Comment:
Please look at this PR to deal with meta tables like nft.trades, bridge.flows, etc...
https://github.com/duneanalytics/spellbook/pull/2212

I don"t think we should duplicate names (bridges_bridge_transfers) in this case, but it's a matter of naming conventions (imo bridge.flows is better than bridges_bridge_transfers here)